### PR TITLE
WIP: TTA sign up design changes 

### PR DESCRIPTION
**Pull request by frankieroberto**
This updates the content and design of the way that candidates can get a teacher training adviser through Apply.

We’ve changed the question about the subject to be more generic so that it also applies to Primary. The answers now use radios instead of an autocomplete, with Primary as the first option, separated by an "or". Some hint text explains why this question is being asked.

### Step 1: section appears in right hand column once candidate is eligible

Screenshot 2023-03-06 at 14 38 52

### Step 2: page describes the teacher training advisers and asks what they’re interested in teaching

screenshot-localhost_3000-2023 03 08-14_17_29

### Step 3: return to application form with success messages

Screenshot 2023-03-06 at 14 39 45

### Step 4: application form shows prompt to speak to the adviser once one is assigned to you

Screenshot 2023-03-06 at 14 40 41

### CODE REVIEW

**Review by MaeveRoseveare — COMMENTED** [review]
A few things from Ellie on GIT

File: app/views/candidate_interface/adviser_sign_ups/new.html.erb [hunk 1]
```diff
@@ -8,11 +8,18 @@
...
+      <p class="govuk-body">Our advisers can answer your questions about how to put together the best application you can. They can help you with:</p>
 
-      <%= render DfE::Autocomplete::View.new(
+      <ul class="govuk-list govuk-list--bullet">
+        <li>writing your personal statement</li>
+        <li>understanding your different training options</li>
+        <li>preparing for interviews</li>
+      </ul>
+
+      <p class="govuk-body">If you continue, we’ll share your personal details and the status of your application with a dedicated advisor. We will not share your personal statement, qualifications or work history with them.</p>
```
[the remark this answers is not in the document]
Comment (frankieroberto) [reply app/views/candidate_interface/adviser_sign_ups/new.html.erb:19]: Oops! 🤦🏻‍♂️

File: config/locales/candidate_interface/adviser_sign_up.yml [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-        ultricies blandit dolor. Cras malesuada, elit quis vulputate semper, ipsum dolor tempus arcu, nec
-        facilisis dolor libero at sapien.
-      submit_text: Sign up
-      call_to_action: Sign up for a TTA
+        Our advisers have years of teaching experience and can give you free, one-to-one help with your application.
+      submit_text: Continue
+      call_to_action: Get an adviser
       flash:
-        success: A Teacher Training Adviser will be in touch to help you with your application.
+        success: An adviser will contact you by email.
```
[the remark this answers is not in the document]
Comment (MaeveRoseveare) [reply config/locales/candidate_interface/adviser_sign_up.yml:10]: My turn for oops.. yes 'within' - in my defence I'm doing this on my clunky DfE surface book while my mac gets sorted. :(

**Comment by frankieroberto** [comment]
Following discussion, we’ve updated the design again to put the question about subject back in, but as radios rather than an autocomplete (to make it easier to scan).

For reference, here’s what it looked like without the subject question:

Screenshot 2023-03-06 at 14 38 58

**Pull request closed without merging**